### PR TITLE
Add Ergon (XRG)

### PR DESCRIPTION
**Pull request by KarolTrzeszczkowski**
Signed-off-by: Karol Trzeszczkowski <karol.trzeszczkowski@gmail.com>

### CODE REVIEW

**Review by tonymorony — CHANGES_REQUESTED** [review]
Hi @KarolTrzeszczkowski ! Thank you for the PR opening.

It looks good, just declaration in `atomic_defi_design/qml/Constants/Style.qml` is missing

You can check this PR as reference: https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1314/

**Comment by KarolTrzeszczkowski** [comment]
@tonymorony thanks for the review. I have a question regarding the PR you linked - what is `atomic_defi_design/qml/Constants/General.qml` for?

**Comment by tonymorony** [comment]
> @tonymorony thanks for the review. I have a question regarding the PR you linked - what is `atomic_defi_design/qml/Constants/General.qml` for?

it's for tradingview charts adding
e.g. this https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/d4147561aa01fc8a4fb6cc1eb1ae4b475cd9ab37/atomic_defi_design/qml/Constants/General.qml#L1120 line adding this tradingview.com/chart/?symbol=BINANCE%3AKMDBTC chart as additional information to the relevant market in DEX:

image

File: assets/config/0.5.2-coins.json [hunk 1]
```diff
@@ -3663,7 +3663,7 @@
     "coinpaprika_id": "tkl-tokel",
     "coingecko_id": "test-coin",
     "electrum": [
-      
+
```
Comment (tonymorony) [inline assets/config/0.5.2-coins.json:3666]: please remove the extra line adding

File: atomic_defi_design/qml/Constants/Style.qml [hunk 2]
```diff
@@ -443,6 +443,7 @@ QtObject {
                                           "XLM": "#737373",
                                           "XMY": "#F01385",
                                           "XRP": "#2E353D",
+                                          "XRG": "#162d50",
```
Comment (tonymorony) [inline atomic_defi_design/qml/Constants/Style.qml:446]: please use capital letter

File: assets/config/0.5.2-coins.json (hunk 1 above)
Comment (KarolTrzeszczkowski) [reply assets/config/0.5.2-coins.json:3666]: Oh, what a mess, I'm sorry this got added. I'm removing.

**Review by tonymorony — APPROVED** [review]

**Pull request merged**